docs/apps/guacamole.md

```diff
@@ -1 +1,93 @@
+
+## Configuring Guacamole
+
+1. Create a `logback.xml` file inside `~/.config/appdata/guacamole/guacamole`
+    * `touch ~/.config/appdata/guacamole/guacamole/logback.xml`
+          or
+    * `sudo nano ~/.config/appdata/guacamole/guacamole/logback.xml`
+2. Open the file with your favorite editor and place the following contents inside of it:
+
+    **NOTE: Make sure to make changes to the timezone accordingly. Check the `php-local.ini` file in `~/.config/appdata/letsencrypt/php` if you are not sure what your timezone is.**
+
+```xml
+<configuration>
+    <!-- Appender for debugging -->
+    <appender name="GUAC-DEBUG" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder>
+            <pattern>%d{HH:mm:ss.SSS, America/New_York} [%thread] %-5level %logger{36} - %msg%n</pattern>
+        </encoder>
+    </appender>
+    <!-- Appender for debugging in a file-->
+    <appender name="GUAC-DEBUG_FILE" class="ch.qos.logback.core.FileAppender">
+        <file>/config/logs/guacd.log</file>
+        <encoder>
+            <pattern>%d{HH:mm:ss.SSS, America/New_York} [%thread] %-5level %logger{36} - %msg%n</pattern>
+        </encoder>
+    </appender>
+    <!-- Log at DEBUG level -->
+    <root level="debug">
+        <appender-ref ref="GUAC-DEBUG"/>
+        <appender-ref ref="GUAC-DEBUG_FILE"/>
+    </root>
+</configuration>
+```
+
+1. Restart the Guacamole container so it creates `guacd.log` in `~/.config/appdata/guacamole/logs/`
+2. Modify your docker-compose.override.yml in `~/.docker/compose/docker-compose.override.yml` file to mount the new log to letsencrypt
+    1. Example:
+
+```yaml
+  letsencrypt:
+    volumes:
+      - ${DOCKERCONFDIR}/guacamole/logs/guacd.log:/var/log/guacamole
+```
+
+   **NOTE: From here on out, we will be using `/var/log/guacamole` to refer to where the guacd.log lives within letsencrypt. This is just an example, you can mount your log file wherever you want inside the letsencrypt container.
+
+1. Recreate your container by running `ds -c up`
+1. Perform an invalid login attempt on Guacamole
+1. Check the new guacd.log file located in `~/.config/appdata/guacamole/logs` to verify the failed login attempt
+    1. Example:
+
+```bash
+grep -i nzbget ~/.config/appdata/guacamole/logs/guacd.log | grep failed
+15:03:17.762 [http-nio-8080-exec-1] WARN  o.a.g.r.auth.AuthenticationService - Authentication attempt from [x.x.x.x, x.x.x.x, x.x.x.x]
+for user "nzbget" failed.
+```
+
+## Configuring F2B
+
+1. Navigate to `~/.config/appdata/letsencrypt/fail2ban`, in there you will see (2) folders `action.d` and `filter.d`, as well as other files, we are going to focus on the file called `jail.local` for now.
+   1. Go ahead and open `jail.local` with your favorite editor as root and copy/paste the following:
+
+```ini
+[guacamole-auth]
+
+enabled = true
+port = http,https
+filter = guacamole-auth
+logpath = /var/log/guacd.log
```

Comment:
This should be `logpath = /var/log/guacamole/guacd.log`

Reply:
Fixed in latest commit.